packages/list/lib/index.ts

```diff
@@ -32,7 +33,7 @@ const args = program.opts()
 autoDetect()
   .list()
   .then(ports => {
-    const formatter = formatters[args.format]
+    const formatter = formatters[args.format as string]
```

Comment:
`opts` can take a generic type argument for the [OptionValues](https://github.com/tj/commander.js/blob/master/typings/index.d.ts#L676) returned by the program:

```ts
const args = program.opts<{ format: string }>()
```

Reply:
Dang, I guess I should have checked the dep version 😅